Makefile

```diff
@@ -22,8 +22,8 @@ build-runtimes:
 	@for e in $(RUNTIMES); do \
 		$(ECHO) "$(MAGENTA)*** Building runtime: $$e...$(OFF)"; \
 		(cd $$e && \
-			cargo build --target x86_64-fortanix-unknown-sgx && \
-			cargo build && \
+			cargo build --target-dir target/x86_64-fortanix-unknown-sgx --target x86_64-fortanix-unknown-sgx && \
```

Comment:
Add comment in the makefile why this is needed and reference the relevant issue.